plugin.py

```diff
@@ -723,6 +723,10 @@ def _do_karma(self, irc, channel, agent, recip, line, explicit=False):
         # Extract 'puiterwijk' out of 'have a cookie puiterwijk++'
         recip = recip.strip().split()[-1]
 
+        # Exclude 'c++', 'g++' or 'i++' (c,g,i), issue #30
+        if len(recip) == 1 and recip.lower() in ['c','g','i']:
```

Comment:
If the first part ``len(recip) == 1`` needed?

At worst to ensure it is not ``None`` you could simply do: ``str(recip).lower() in ...`` :)